doc/03-Configuration.md

```diff
@@ -25,9 +21,27 @@ that is used as the sender's address (From) in E-mails.
 
 There are four permissions that can be used to control what can be managed by whom.
 
-Permission           | Applies to
----------------------|----------------
-reporting/reports    | Reports (create, edit, delete)
-reporting/schedules  | Schedules (create, edit, delete)
-reporting/templates  | Templates (create, edit, delete)
-reporting/timeframes | Timeframes (create, edit, delete)
+| Permission           | Applies to                        |
+|----------------------|-----------------------------------|
+| reporting/reports    | Reports (create, edit, delete)    |
+| reporting/schedules  | Schedules (create, edit, delete)  |
+| reporting/templates  | Templates (create, edit, delete)  |
+| reporting/timeframes | Timeframes (create, edit, delete) |
+
+## Icinga Reporting Daemon
```

Comment:
This section needs a little adjustment. Packages will install the systemd service automatically. And I would add `--now` to the `systemctl enable` command to start the daemon right away as it's periodically reloading the configuration anyway.